src/query.js

```diff
@@ -8,8 +8,8 @@ import { equals } from './utils.js'
  * This is a parquet-aware query engine that can read a subset of rows and columns.
  * Accepts optional filter object to filter the results and orderBy column name to sort the results.
  * Note that using orderBy may SIGNIFICANTLY increase the query time.
- *
- * @param {ParquetReadOptions & { filter?: ParquetQueryFilter, orderBy?: string }} options
+ * @import {BaseParquetReadOptions} from '../src/types.d.ts'
+ * @param {BaseParquetReadOptions & { filter?: ParquetQueryFilter, orderBy?: string }} options
```

Comment:
An alternative syntax for the same result. Maybe more explicit:


```suggestion
 * @import {ParquetReadOptions} from '../src/types.d.ts'
 * @param {Omit<ParquetReadOptions, 'rowFormat' | 'onComplete'> & { filter?: ParquetQueryFilter, orderBy?: string }} options
```